roles/dosomething.base/tasks/install-common-tools.yml

```diff
@@ -12,3 +12,8 @@
     - screen
     - bind9utils
     - sysstat
+
+- name: Disable AppArmor
+  sudo: yes
+  apt: name=apparmor state=absent
+  notify: restart server
```

Comment:
As I understand, we also need to add a local action to wait for server to be back. Let me find an example.